db/gen_fixtures.rb

```diff
@@ -0,0 +1,67 @@
+require 'csv'
```

Comment:
@iandouglas @AliSchlereth This is already merged, but where should this file live so I can move it in a future commit? It just generates fixture data from the full (large) CSV files to make the fixture data managable so that Heroku will accept it when we seed. Is it okay to have it live in `db/`? Or is there a better place to put this? Or just add it to the .gitignore and forget it entirely?

Reply:
Check out this portion of the spec https://github.com/turingschool/backend-curriculum-site/blob/gh-pages/module2/projects/bike-share-redux.md#details-for-seeding-csvs Let us know if you have further questions and/or this section isn't clear.